app/controllers/api/allocations_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       allocations_params = Allocations::ParamsValidator.new(filter_params, sort_params)
       if allocations_params.valid?
         paginate allocations, serializer: AllocationsSerializer, include: included_relationships, fields: AllocationsSerializer::INCLUDED_FIELDS do |paginated_allocations, options|
-          options[:params] = paginated_allocations.move_totals
+          options[:params] = { totals: paginated_allocations.move_totals }
```

Comment:
This is just namespacing the existing `params` to avoid a clash with new `included` param.